tutorials/cap-extend-sfsf-intro/cap-extend-sfsf-intro.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Introduction and Preparation
```

Comment:
This title is very ambiguous.  It is also not stated in an active tone.  We do not know for what the introduction and preparation are for.  Perhaps something like "Prepare to Develop the SAP SuccessFactors Extension"?